fastlane/helpers/github.rb

```diff
@@ -0,0 +1,10 @@
+def create_release_management_pull_request(base_branch, title)
+  create_pull_request(
+      api_token: ENV['GITHUB_TOKEN'],
+      repo: GHHELPER_REPO,
+      title: title,
+      head: Fastlane::Helper::GitHelper.current_git_branch,
```

Comment:
Fastlane defaults to the current branch. However, it does so by using the [`git_branch`](https://docs.fastlane.tools/actions/git_branch/#git_branch) helper which uses environment variables to override the value. We do not want this behavior which is why we added our own helper in https://github.com/wordpress-mobile/release-toolkit/pull/463.